### ISSUE TITLE
Accessibility: google translate widget

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#206]
Hello, I am running into accessibility issues for the google translate widget. I'm assuming these are things that can't be changed on our end and would have to be updated by Google?

One issue is "Presentational attributes are used"
`<iframe frameborder="0" class="goog-te-menu-frame skiptranslate" title="Language Translate Widget" style="visibility: visible; box-sizing: content-box; width: 1030px; height: 263px; display: none;"></iframe>`

Information on how to fix:
Presentational attributes such as 'border', 'align', or 'bgcolor' are used. CSS should be used for styling instead.

It's important to review these issues and determine if they can be fixed by moving the presentational attributes to your CSS. It's not always practical to make these changes, but should be made when possible.

**Comment by joelworsham** [comment]
Yeah this one's on Google, but I can definitely say that using "Presentation attributes" is _not_ always bad. I'm guessing this "error" shown is more to alert you to it, rather than specifying to remove it? Either way, I would consider this a non-issue.

**Comment by banksashleym** [comment]
That what I figured :) Thanks for the info!
